scripts/xtensa-build-zephyr.py

```diff
@@ -1169,6 +1175,29 @@ def reproducible_checksum(platform, ri_file):
 	chk256 = sof_ri_info.EraseVariables(ri_file, parsed_ri, west_top / repro_output)
 	print('sha256sum {0}\n{1} {0}'.format(repro_output, chk256))
 
+def create_tarball():
+	# vendor directories (typically under /lib/firmware) to include
+        # in the tarball
+	vendor_dirs = [ 'intel']
+
+	build_top = west_top / 'build-sof-staging' / 'sof'
```

Comment:
use path join https://www.geeksforgeeks.org/python-os-path-join-method/

Reply:
I'm afraid I disagree: `pathlib` is the newer and shinier Python API.

Reply:
Appears like im out of date on my python best practices